mmengine/visualization/visualizer.py

```diff
@@ -82,6 +83,8 @@ class Visualizer(ManagerMixin):
             Defaults to empty dict.
         fig_show_cfg (dict): Keyword parameters of figure for showing.
             Defaults to empty dict.
+        backend (str): Draw backend config. It can be 'matplotlib' or 'cv2'.
+            Defaults to 'matplotlib'.
```

Comment:
```suggestion
        backend (str): Draw backend config. It can be 'matplotlib' or 'cv2'.
            Defaults to 'matplotlib'.
            `New in version 0.8.5.`
```